fhir-converter/src/routes.js

```diff
@@ -22,7 +22,9 @@ var handlebarsHelpers = require("./lib/handlebars-converter/handlebars-helpers")
 var ncp = require("ncp").ncp;
 
 module.exports = function (app) {
-  const workerPool = new WorkerPool("./src/lib/workers/worker.js", require("os").cpus().length);
+  const amountOfWorkers = require("os").cpus().length;
+  console.log(`Creating a pool of ${amountOfWorkers} workers (${require("os").cpus().length} vCPUs)`)
```

Comment:
nit: can use `amountOfWorkers` twice here instead of doing `${require("os").cpus().length}` again

Reply:
I intentionally left separated bc we could configure the amount of workers to a number smaller than the amount of cpus